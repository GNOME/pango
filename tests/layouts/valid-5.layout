{
  "text" : "A test with multiple paragraphs and with no-break attributes, which might trigger a crash.\nIf it doesn't the fix has worked.\n",
  "attributes" : [
    {
      "start" : 21,
      "end" : 31,
      "type" : "font-features",
      "value" : "tnum=1"
    }
  ],
  "font" : "Cantarell 11",
  "width" : 194560
}
