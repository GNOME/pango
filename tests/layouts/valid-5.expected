A test with multiple paragraphs and with no-break attributes, which might trigger a crash.
If it doesn't the fix has worked.

--- parameters

wrapped: 1
ellipsized: 0
lines: 7
width: 194560

--- attributes

range 0 21
range 21 31
[21,31]font-features=tnum=1
range 31 2147483647

--- directions

0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 

--- cursor positions

0(0) 1(0) 2(0) 3(0) 4(0) 5(0) 6(0) 7(0) 8(0) 9(0) 10(0) 11(0) 12(0) 13(0) 14(0) 15(0) 16(0) 17(0) 18(0) 19(0) 20(0) 20(1) 22(0) 23(0) 24(0) 25(0) 26(0) 27(0) 28(0) 29(0) 30(0) 31(0) 32(0) 33(0) 34(0) 35(0) 36(0) 37(0) 38(0) 39(0) 40(0) 41(0) 42(0) 43(0) 43(1) 45(0) 46(0) 47(0) 48(0) 49(0) 50(0) 51(0) 52(0) 53(0) 54(0) 55(0) 56(0) 57(0) 58(0) 59(0) 60(0) 61(0) 62(0) 63(0) 64(0) 65(0) 66(0) 67(0) 67(1) 69(0) 70(0) 71(0) 72(0) 73(0) 74(0) 75(0) 76(0) 77(0) 78(0) 79(0) 80(0) 81(0) 82(0) 83(0) 84(0) 85(0) 86(0) 87(0) 88(0) 89(0) 89(1) 91(0) 92(0) 93(0) 94(0) 95(0) 96(0) 97(0) 98(0) 99(0) 100(0) 101(0) 102(0) 103(0) 104(0) 105(0) 106(0) 107(0) 108(0) 109(0) 110(0) 111(0) 112(0) 113(0) 114(0) 115(0) 116(0) 116(1) 118(0) 119(0) 120(0) 121(0) 122(0) 123(0) 123(1) 125(0) 

--- lines

i=1, index=0, paragraph-start=1, dir=ltr 'A test with multiple '
i=2, index=21, paragraph-start=0, dir=ltr 'paragraphs and with no-'
i=3, index=44, paragraph-start=0, dir=ltr 'break attributes, which '
i=4, index=68, paragraph-start=0, dir=ltr 'might trigger a crash.
'
i=5, index=91, paragraph-start=1, dir=ltr 'If it doesn't the fix has '
i=6, index=117, paragraph-start=0, dir=ltr 'worked.
'
i=7, index=125, paragraph-start=1, dir=ltr ''

--- runs

i=1, index=0, chars=21, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'A test with multiple '
[21,31]font-features=tnum=1
i=2, index=21, no run, line end
i=3, index=21, chars=23, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'paragraphs and with no-'
[21,31]font-features=tnum=1
i=4, index=44, no run, line end
i=5, index=44, chars=24, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'break attributes, which '
[21,31]font-features=tnum=1
i=6, index=68, no run, line end
i=7, index=68, chars=22, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'might trigger a crash.'
[21,31]font-features=tnum=1
i=8, index=90, no run, line end
i=9, index=91, chars=26, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'If it doesn't the fix has '
i=10, index=117, no run, line end
i=11, index=117, chars=7, level=0, gravity=south, flags=0, font=OMITTED, script=latin, language=en-us, 'worked.'
i=12, index=124, no run, line end
i=13, index=125, no run, line end
